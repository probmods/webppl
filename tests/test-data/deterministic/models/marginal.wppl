var m1 = Infer({method: 'forward', samples: 1, model: constF(0)});
var m2 = Infer({method: 'forward', samples: 2, model: constF(0)});
[
  m1.support(),
  m2.support()
];
