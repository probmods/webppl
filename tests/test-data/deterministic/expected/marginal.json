{
    "result": [[0], [0]]
}
